pages/common/file.md

```diff
@@ -0,0 +1,16 @@
+# file
+
+> Determine file type
+
+- Give a description of the type of the specified file
+
+`file {{readme}}`
+`file {{python3.so.1}}`
```

Comment:
One example should suffice, and preferably using a generic parameter name such as `{{filename}}`. I get that you're trying to make the point that the extension is unnecessary, but maybe it's better to say it explicitly in the example description.